.bazeliskrc

```diff
@@ -0,0 +1,2 @@
+BAZELISK_BASE_URL=https://github.com/aspect-build/aspect-cli/releases/download
+USE_BAZEL_VERSION=aspect/5.9.36
```

Comment:
Let's ommit the whole .bazeliskrc file, we don't know if we want to use the aspect way of linting.